client/src/pages/logs-screen/LogTable.tsx

```diff
@@ -88,67 +95,72 @@ const LogTable: React.FC<LogTableProps> = ({ selectedLogType }) => {
             </Tr>
           </Thead>
           <Tbody>
-            {currentData.map((item: LogItemInf, index: number) => (
-              <Tr
-                key={index}
-                onClick={() => {
-                  navigate(`/logdetails/${item.logId}`);
-                }}
-              >
-                <Td>
-                  <Text color="teal" fontWeight={"bold"}>
-                    {item.logId}
-                  </Text>
-                  <Text mt="1" mb="2">
-                    {" "}
-                    {item.logText}
-                  </Text>
-
-                  <Button
-                    px="2"
-                    size="xs"
-                    display="flex"
-                    gap="2"
-                    alignItems="center"
-                  >
-                    <span>
-                      {" "}
-                      <FaFolder />
-                    </span>
-                    <span>{item.project.projectCode}</span>
-                  </Button>
-                </Td>
-                <Td>
-                  {" "}
-                  <Badge
-                    variant="solid"
-                    colorScheme={
-                      item.logType === "ERROR"
-                        ? "red"
-                        : item.logType === "WARNING"
-                        ? "yellow"
-                        : item.logType === "INFO"
-                        ? "blue"
-                        : "gray"
-                    }
-                  >
-                    {item.logType}
-                  </Badge>
-                </Td>
-
-                <Td fontWeight="bold">{item.created_at}</Td>
-                <Td>
-                  <Text mb="2" fontWeight="bold">
-                    {item.metaTimestamp ? item.metaTimestamp : "N/A"}
-                    {/* @Todo currently static, once added to Api it will be made dynamic */}
-                  </Text>
-                  <Text>
-                    {item.metadescription ? item.metadescription : "N/A"}
-                    {/* @Todo currently static, once added to Api it will be made dynamic */}
-                  </Text>
+            {currentData.length === 0 ? (
+              <Tr>
+                <Td colSpan={4} textAlign="center">
+                  No items found
                 </Td>
               </Tr>
-            ))}
+            ) : (
+              currentData.map((item: LogItemInf, index: number) => (
+                <Tr
+                  key={index}
+                  onClick={() => {
+                    navigate(`/logdetails/${item.logId}`);
+                  }}
+                >
+                  <Td>
+                    <Text color="teal" fontWeight="bold">
+                      {item.logId}
+                    </Text>
+                    <Text mt="1" mb="2">
+                      {" "}
```

Comment:
`{" "}` Why this is given in many lines. How much difference is there with this

Reply:
Removed it..It was Automatically Generating when i save the file